### ISSUE TITLE
A11y_AzureDataStudio_Filter_SortingAscending/Descending_ScreenReader: Orca is announcing incorrect control type for the "Search" edit control.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#16024]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11yMAS;#A11yTCS;A11ySev2;#SH-AzureDataStudio-Linux-Jul2021;#SQL Azure Data Studio;#Linux;#DesktopApp;#WCAG4.1.2;#Orca;#AINotInScope;#FTP;

### Environment Details:
Azure Data Studio Version: 1.31.0-insiders
OS Name: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
Screen Reader: Orca

### Steps to Reproduce:

1. Launch Azure Data Studio
2. Connect to any SQL server and write a query
3. TAB till results grid and press F3 so that the filters pop-up will open
4. Start Orca screen reader and then TAB to "Search" edit control
5. Observe the announcement by screen reader

### Actual:
The screen reader announces "Search entry" for "Search" edit control which is incorrect as there is no information announced for the control type of the search control.

### Expected:
The screen reader should announce "Search edit "

### User Impact:
Screen reader users will get confused with the control type of the button.

**Comment by alanrenmsft** [comment]
@12-shweta does ORCA read anything as "edit"? I am afraid that is an Orca thing to read text box as `entry`.

**Comment by 12-shweta** [comment]
After discussion with SME this seem to just be the differences in how Orca announces these elements

**Comment by 12-shweta** [comment]
Github Tags: #Closed;
